yang/IETF-04/ietf-vpn-common.yang

```diff
@@ -0,0 +1,505 @@
+module ietf-vpn-common {
+  
+  yang-version 1.1;
+  namespace "urn:ietf:params:xml:ns:yang:ietf-vpn-common";
+  prefix vpn-common;
+
+  import ietf-netconf-acm {
+    prefix nacm;
+    reference
+      "RFC 8341: Network Configuration Access Control Model";
+  }
+  import ietf-routing-types {
+    prefix rt-types;
+    reference
+      "RFC 8294: Common YANG Data Types for the Routing Area";
+  }
+  import ietf-yang-types {
+    prefix yang;
+    reference
+      "Section 3 of RFC 6991";
+  }
+  organization
+    "IETF OPSA (Operations and Management Area) Working Group ";
+  contact
+    "WG Web:   <http://tools.ietf.org/wg/opsawg/>
+        WG List:  <mailto:opsawg@ietf.org>
+        Author:    Samier Barguil
+                  <mailto:samier.barguilgiraldo.ext@telefonica.com>
+        Editor:    Oscar Gonzalez de Dios
+                  <mailto:oscar.gonzalezdedios@telefonica.com>
+        Author:   Mohamed Boucadair
+                  <mailto:mohamed.boucadair@orange.com>           
+        Author:    Luis Angel Munoz
+                  <mailto:luis-angel.munoz@vodafone.com>
+        Author:    Alejandro Aguado
+                  <mailto:alejandro.aguado_martin@nokia.com> 
+    ";
+  description
+    "This YANG module defines a generic network-oriented model
+     for the configuration of Layer 3 Virtual Private Networks.
+     Copyright (c) 2020 IETF Trust and the persons identified as
+     authors of the code.  All rights reserved.
+
+     Redistribution and use in source and binary forms, with or
+     without modification, is permitted pursuant to, and subject to
+     the license terms contained in, the Simplified BSD License set
+     forth in Section 4.c of the IETF Trust's Legal Provisions
+     Relating to IETF Documents
+     (https://trustee.ietf.org/license-info).
+
+     This version of this YANG module is part of RFC XXXX
+     (https://www.rfc-editor.org/info/rfcXXXX); see the RFC itself
+     for full legal notices.";
+
+  revision 2020-06-03 {
+    description
+      "Initial revision.";
+    reference
+      "RFC XXXX: A VPN common types YANG Model";
+      // RFC Ed.: replace XXXX with actual RFC number and remove
+     // this note
+  }
+
+/* Feature */
```

Comment:
Shouldn't we include "common" features between L3SM/L3SN as well?

Reply:
Yes, I will start with it.

Reply:
My key question is @oscargdd @boucadair may I upload the L2NM in this repo?